Source/Models/BrickModels.swift

```diff
@@ -17,18 +17,36 @@ public class Brick: CustomStringConvertible {
     /// Identifier of the brick. Defaults to empty string
     public let identifier: String
 
+    public var size: BrickSize
+    
     /// Width dimension used to calculate the width. Defaults to .Ratio(ratio: 1)
-    public var width: BrickDimension
-
+    @available(*, deprecated, message="use size.width")
```

Comment:
I would not have this `@deprecated` message. Width and height are just computed properties (like they are for `CGRect`)